lasagne/layers/recurrent.py

```diff
@@ -976,20 +973,14 @@ def step(input_n, cell_previous, hid_previous, W_hid_stacked,
             cell = forgetgate*cell_previous + ingate*cell_input
 
             if self.peepholes:
-                outgate += cell*W_cell_to_outgate
+                outgate += cell*self.W_cell_to_outgate
 
             # Compute new hidden unit activation
             hid = outgate*self.nonlinearity(cell)
             return [cell, hid]
 
-        def step_masked(input_n, mask_n, cell_previous, hid_previous,
-                        W_hid_stacked, W_cell_to_ingate, W_cell_to_forgetgate,
-                        W_cell_to_outgate, W_in_stacked, b_stacked):
-
-            cell, hid = step(input_n, cell_previous, hid_previous,
-                             W_hid_stacked, W_cell_to_ingate,
-                             W_cell_to_forgetgate, W_cell_to_outgate,
-                             W_in_stacked, b_stacked)
+        def step_masked(input_n, mask_n, cell_previous, hid_previous, *args):
+            cell, hid = step(input_n, cell_previous, hid_previous, args)
```

Comment:
It works because you're not using it, but you should pass `*args` instead of `args` (just in case the original step function decides to use `*args` in future).